README.md

```diff
@@ -31,20 +31,30 @@ The Javadocs are in the `./javadoc` directory.
 
 ## Relationship to the Iceberg Repository
 
-The `documentation` folder in the [Iceberg repository](https://github.com/apache/iceberg) contains all the markdown docs used by the website.
-The `common` subfolder contains contents used by the landing page.
-The `versioned` subfolder contains the contents different for each version.
+All markdown pages that are part of a versioned released are maintained in the iceberg repository. All pages common across all version releases are
+kept here in the iceberg-docs repo. A few exceptions are the markdown files that can be found in the `format` folder in the iceberg repository and contains
+markdown files that are copied to `./landing-page/content/common/format`.
+
+`apache/iceberg`
+- The `docs` folder in the [Iceberg repository](https://github.com/apache/iceberg) contains all the markdown docs used by the **versioned** docs site.
+- The `format` folder contains some items that are common accross all versions, such as the Iceberg format specification.
+
+`apache/iceberg-docs`
+- The `docs/content/docs` folder is a mirror of the `docs` folder in the iceberg repo and the target when copying the docs over during a release
+- The `landing-page/content/common` folder is where you can find the common markdown files shared across all versions
 
 During each new release, the release manager copies contents from the Iceberg repository to this docs repository and cuts a new version branch.
-Contents under `common` are copied to `./landing-page/contents/common` here,
-and contents under `versioned` are copied to `./docs/contents/docs` here.
+Contents under `docs` are copied to `./docs/content/docs` here,
```

Comment:
Are these docs used in the main site build?

If not, can we make this an empty directory except for in release branches? That is, I think the current flow is this:

1. Copy all docs from the iceberg repo into `docs/contents/docs` and commit to main
2. Create a branch from main named for the release version
3. Update the latest branch head to point to the release branch

Instead, could we do this?

1. Create a branch from main named for the release version
2. Copy the iceberg repo docs into the release branch under `docs/contents/docs`
3. Update the latest branch head to point to the release branch

That would mean that `main` never has a copy of versioned docs and we can say that you can edit a markdown file wherever you find it.

And we could have a more specific README in the `docs/contents/docs` folder.

Reply:
Updated the instructions to say:

> During each new release, the release manager will:
> 1. Create a branch in this repo from main named for the release version
> 2. Copy the contents under `docs` in the iceberg repo to `./docs/content/docs` in the release branch
> 3. Copy the contents under `format` in the iceberg repo to `./landing-page/content/common/format` in the release branch
> 4. Generate the javadocs for the release and copy them into the `javadoc` directory in the release branch
> 5. Update the latest branch HEAD to point to the release branch HEAD